http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -874,18 +838,15 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
       autoscalingConfig = None
     )
 
-    // regionName can be empty in some test configurations
-    val regionName = if (azureRegionOpt.isEmpty) "" else azureRegionOpt.map(_.name).getOrElse("")
     for {
       nodepool <- defaultNodepool
       defaultClusterName <- KubernetesNameUtils.getUniqueName(KubernetesClusterName.apply)
-      clusterName = aksClusterName.map(c => KubernetesClusterName(c.value)).getOrElse(defaultClusterName)
     } yield SaveKubernetesCluster(
       cloudContext = cloudContext,
-      clusterName = clusterName,
+      clusterName = defaultClusterName,
       location = config.leoKubernetesConfig.clusterConfig.location,
       region =
-        if (cloudContext.cloudProvider == CloudProvider.Azure) RegionName(regionName)
+        if (cloudContext.cloudProvider == CloudProvider.Azure) RegionName("unset")
```

Comment:
So this feels like this will break the UI, and especially the work that @cindy-broadinstitute did recently. This means that the UI won't be able to display the app region anymore

Reply:
Yes, this will definitely break the UI. The RegionName (Region for Azure) is saved to the config here and used by the UI.

Reply:
OK makes sense -- will try to fix.

It would be nice if we could pull the region from the WSM workspace (which we do still query in Front Leo). Will see if that's possible -- failing that, we could asynchronously update this column when we query the LZ in back leo.

Reply:
I think UI probably knows abt the region since it calls WSM anyway...for future improvements tho

Reply:
I don't think pulling region from workspace at this point is viable, since there was a feature request to persist region at the app level

Reply:
Haven't pushed yet, but I am planning to pull the region from the WSM VM resource in back Leo.